src/o365/spo/commands/SpoContext.ts

```diff
@@ -0,0 +1,3 @@
+export interface SpoContext {
```

Comment:
I moved this interface within the get.ts file. Let's keep it private to the `spo get` command until there is a need to share it with a different command.